src/content/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui.mdx

```diff
@@ -0,0 +1,77 @@
+
+[Synthetics private locations](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations/) must have [synthetics job manager](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide/) installed to run monitors using the Chrome 100+ or Node 16.10 runtimes. 
+
+## Proactive upgrade testing [#proactive]
+
+We will test all synthetic monitors using legacy runtimes against our latest runtimes in a proactive way. This testing:
+
+* It happens once and will only trigger again if someone updates the monitor.
+* Is used only to populate the runtime upgrade UI.
+* Doesn't [consume synthetic checks](/docs/synthetics/synthetic-monitoring/getting-started/monitor-limits/#edition-limits).
+* Doesn't populate the results UI for the monitor.
+* Doesn't create `SyntheticCheck`, `SyntheticRequest`, or any other job artifacts.
+
+Each upgrade test result will populate the runtime upgrade UI in one of three tabs:
+
+<Collapser
+  id="passed"
+  title="Passed"
+>
+
+This group of monitors completed successfully when tested in our latest runtimes. We recommend using the [Upgrade all button](#upgrade) to upgrade all of these monitors to the new runtime.
+
+</Collapser>
+
+<Collapser
+  id="failed"
+  title="Failed"
+>
+
+This group of monitors failed when tested in our latest runtimes. This can happen if the monitor is failing in the legacy runtime. Or, it may be due to a script update due to incompatibility between the runtimes. While we have provided as much backward compatibility as possible, there are a few breaking changes.
+
+We recommend [troubleshooting](/docs/synthetics/synthetic-monitoring/troubleshooting/runtime-upgrade-troubleshooting/) and [upgrading these monitors individually](/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime/#runtime-convert). You can do this by changing the runtime dropdown in the monitor configuration. Then, use validate to test the monitor while making script updates.
+
+<Callout variant="tip">
+  We don't recommend the upgrade of all failed monitors.
+</Callout>
+
+</Collapser>
+
+
+<Collapser
+  id="blocked"
+  title="Blocked"
+>
+
+We could not test this group of monitors with the latest runtime. This happens when [setting up the monitor for a private location that does not yet use the synthetics job manager](#requirements). Once you install a synthetics job manager, it will trigger the runtime upgrade testing for these monitors. Once that testing is complete, the monitors will move from the blocked tab to the [passed](#passed) or [failed](#failed) tab.
+
+Forcing the upgrade of blocked monitors from the runtime upgrade UI is not permitted. Doing so would prevent the monitors from running.
```

Comment:
is not > isn't. Also probably change this to say "We don't allow" instead of "is not permitted", if that makes sense

Reply:
yes, you're right. I've changed it to this:
`You can't force the upgrade of locked monitors from the runtime upgrade UI. This would prevent the monitors from running.`